docs/ROADMAP.md

```diff
@@ -1,53 +1,89 @@
-make them simpler to use and more flexible. We are looking at options for
-making setters more declarative, in particular making it possible to update
-the setter values directly in a file and then propagate all values into the
-resources in the package. We have also seen that substitutions don't always
-work well for large numbers of resources, so we are looking to make them simpler
-and more flexible. We are also introducing a Search and Replace functionality
-that will make bulk edits across manifests much easier.
-
-## Functions
-Kpt functions in their current format are useful for performing individual
-tasks, but they are hard to leverage for more complex use-cases such as
-hydrating DRY configuration or generating variants of a package for different
-environments. We are looking to introduce the concept of a pipeline of kpt
-functions as an optional part of a kpt package. A pipeline declares operations
-that will be performed on the resources declared within the package as well on
-other packages using recursive resolution. It will also provide flexibility to
-output the results of the operations using different sinks modes: in-place,
-stdout, or an external directory. 
-
-## Live
-_kpt live_ is currently using a ConfigMap to store inventory information. This is
-convenient as it doesn’t require any CRDs, but it has challenges around encoding
-of the GroupKind, name and namespace, and it doesn’t allow us to easily add
-additional metadata about a package, such as the package version. We are working
-on migrating kpt to use a CRD for storing the inventory information.
-
-We have also started looking at how to best handle nested packages during apply.
-Currently the nested structure is not reflected in the live state on the
-apiserver. 
-
-The majority of the code for kpt live is in the cli-utils repo. Our
+Last updated: June 1st, 2021
+
+Draft of the *v1* release slated for Summer 2021.
+
+### 1. Declarative function pipeline
+
+kpt has added a declarative way to customize and validate configuration.  
+This allows you to run several mutation and validation 
+functions in a pipeline alleviating the need to create shell scripts that do 
+the same thing.  Further information can be found in the 
+[declarative function execution]  section of the Kpt Book.
+
+### 2. Setters
+
+Setters used to be a special entity without sufficient differentiation from
+KRM functions. In kpt v1 setters become just another function and drastically
+simplify syntax.  Configuring 4 setters used to take 20 lines of yaml 
+and now takes 6.  Setters also get recursively applied to sub-packages by
+default.  For further information on the setter function please visit: 
+[apply-setters documentation]. 
+
+### 3. Resource merge
+
+kpt package updates now default to the resource merge strategy 
+which allows you to edit configuration with an text editor of your choice 
+and still be able to rebase with upstream changes. 
```

Comment:
We should say `merge`, not `rebase`.

Reply:
merge and merge... I was trying to avoid that.  I will use "get updates"